src/test-output.xml

```diff
@@ -0,0 +1,3 @@
+<test-run id="2" testcasecount="1" result="Passed" start-time="2023-08-07 14:31:05.568996" end-time="2023-08-07 14:31:10.467617" duration="4" total="1" passed="1" failed="0" inconclusive="0" skipped="0" asserts="0" clr-version="3.10.2 (main, Mar 15 2022, 16:57:02) [GCC 9.3.0]" engine-version="3.6.2"><command-line>/home/arcvmware/Code/github.com/azure/.venv/lib/python3.10/site-packages/pytest/__main__.py -x -v --forked -p no:warnings --log-level=WARN --junit-xml /home/arcvmware/.azdev/env_config/home/arcvmware/Code/github.com/azure/.venv/test_results.xml /home/arcvmware/Code/github.com/azure/azure-cli-extensions/src/connectedvmware/azext_connectedvmware/tests/latest -n auto</command-line><filter><test re="0">/home/arcvmware/Code/github.com/azure/azure-cli-extensions/src/connectedvmware/azext_connectedvmware/tests/latest</test></filter><test-suite id="azext_connectedvmware/tests/latest/test_connectedvmware_scenario.py" name="azext_connectedvmware/tests/latest/test_connectedvmware_scenario.py" fullname="azext_connectedvmware/tests/latest/test_connectedvmware_scenario.py" methodname="" classname="" runstate="Runnable" type="Assembly" testcasecount="1" result="Passed" label="" start-time="2023-08-07 14:31:10.449712" end-time="2023-08-07 14:31:10.467224" duration="0.017512" asserts="0" total="1" passed="1" failed="0" warnings="0" inconclusive="0" skipped="0"><properties><property name="python_version" value="3.10.2 (main, Mar 15 2022, 16:57:02) [GCC 9.3.0]" /></properties><test-case id="100" name="azext_connectedvmware/tests/latest/test_connectedvmware_scenario.py::ConnectedvmwareScenarioTest::test_connectedvmware" fullname="azext_connectedvmware/tests/latest/test_connectedvmware_scenario.py::ConnectedvmwareScenarioTest::test_connectedvmware" methodname="test_connectedvmware" classname="ConnectedvmwareScenarioTest" runstate="Runnable" seed="1" result="Passed" label="" start-time="2023-08-07 14:31:10.449712" end-time="2023-08-07 14:31:10.467224" duration="0.017512" asserts="0"><properties><property name="python-version" value="3.10.2 (main, Mar 15 2022, 16:57:02) [GCC 9.3.0]" /><property name="fspath" value="azext_connectedvmware/tests/latest/test_connectedvmware_scenario.py" /></properties><environment framework-version="3.6.2" clr-version="3.10.2 (main, Mar 15 2022, 16:57:02) [GCC 9.3.0]" os-version="5.17.5-76051705-generic" platform="Linux" cwd="/home/arcvmware/Code/github.com/azure/azure-cli-extensions/src" machine-name="x86_64" user="" user-domain="" culture="en_IN" uiculture="en_IN" os-architecture="64bit" /><failure><message><![CDATA[[gw0] linux -- Python 3.10.2 /home/arcvmware/Code/github.com/azure/.venv/bin/python]]></message><stack-trace><![CDATA[None]]></stack-trace></failure><reason><message><![CDATA[WARNING  msrest.serialization:serialization.py:187 Readonly attribute name will be ignored in class &lt;class 'azext_connectedvmware.vendored_sdks.hybridcompute.models._models_py3.MachineExtension'&gt;
```

Comment:
Why are there two `test-output.xml` files and one is in the root directory of extension repo?

Reply:
These might have been committed by mistake. Maybe they got created when I ran azdev test from a different folder or because there were two `azdev extension repo` during development. I have removed these files.